include/timers.h

```diff
@@ -1361,6 +1361,20 @@ BaseType_t xTimerGenericCommand( TimerHandle_t xTimer,
 
 #endif
 
+#if  ( configUSE_DAEMON_TASK_STARTUP_HOOK != 0 )
```

Comment:
```suggestion
#if ( configUSE_DAEMON_TASK_STARTUP_HOOK != 0 )
```

Reply:
Suggest using `clang-format` instead of `uncrustify` if you want to enforce that level of formatting requirements.

Reply:
Changed.